src/Strategy/Googlesource/RepoManifest.hs

```diff
@@ -71,11 +71,14 @@ mkProject project =
     , projectLicenses = pure []
     }
 
-getDeps :: (Has ReadFS sig m, Has Diagnostics sig m) => RepoManifestProject -> m (Graphing Dependency)
+getDeps :: (Has ReadFS sig m, Has Diagnostics sig m) => RepoManifestProject -> m (Graphing Dependency, GraphBreadth)
 getDeps = analyze' . repoManifestXml
 
-analyze' :: (Has ReadFS sig m, Has Diagnostics sig m) => Path Abs File -> m (Graphing Dependency)
-analyze' file = buildGraph <$> nestedValidatedProjects (parent file) file
+-- TODO confirm this is Partial @scott
+analyze' :: (Has ReadFS sig m, Has Diagnostics sig m) => Path Abs File -> m (Graphing Dependency, GraphBreadth)
+analyze' file = do
+  graph <- buildGraph <$> nestedValidatedProjects (parent file) file
+  pure (graph, Partial)
```

Comment:
cc @spatten

Reply:
I haven't touched this code in a long time, and it didn't look like this when I wrote it :). 

I think @cnr changed this to make it output a `Graphing Dependency` back in October.

Reply:
The underlying strategy probably hasn't changed, just the scaffolding around it.  Was it originally outputting a partial graph or a complete graph.  i.e.: would it benefit from recursive analysis of the deps discovered?

Reply:
I think it's outputting a complete graph.

@kitified is that correct?

Reply:
@spatten I am not familiar with the `Googlesource` strategy 😬

Reply:
Interesting. I think this was originally built as part of AOSP parsing, and it might be dead code. I'm definitely not sure on that, though

I'll take a look and refresh my memory and report back here

Reply:
I did some reading and remembering, and I can confirm that this is a partial graph.

I'm 90% sure that this is dead code, but that doesn't have anything to do with this PR.

So :+1: from me on the `Partial` being correct